tools/ovc/openvino/tools/ovc/cli_parser.py

```diff
@@ -559,16 +559,21 @@ def get_model_name_from_args(argv: argparse.Namespace):
     if hasattr(argv, 'output_model') and argv.output_model:
         model_name = argv.output_model
 
-        if not os.path.isdir(argv.output_model):
+        if not os.path.isdir(argv.output_model) and not argv.output_model.endswith(os.sep):
+            # In this branch we assume that model name is set in 'output_model'.
             if not model_name.endswith('.xml'):
                 model_name += '.xml'
+            # Logic of creating and checking directory is covered in save_model() method.
```

Comment:
Should we update documentation for `ovc` and `convert_model` in this case?

Reply:
convert_model() is not affected by these changes, as `get_model_name_from_args()` is used only in cli tool:
https://github.com/openvinotoolkit/openvino/blob/8034d1795f8ce4d149c6d480a4056f80e014dde6/tools/ovc/openvino/tools/ovc/main.py#L25

Regarding OVC cli tool, I updated the cli tool help to clarify current logic of output_dir, as it looked not informative.
In docs we have description here: https://github.com/openvinotoolkit/openvino/blob/8034d1795f8ce4d149c6d480a4056f80e014dde6/docs/articles_en/documentation/openvino_legacy_features/mo_ovc_transition.md?plain=1#L35
It looks correct.